package.json

```diff
@@ -21,6 +21,7 @@
     "glob": "^5.0.14",
     "iferr": "^0.1.5",
     "mkdirp": "^0.5.1",
+    "options": "0.0.6",
```

Comment:
Simply used `npm install --save options` and it gave me this.  Looks like the latest version of [options](https://github.com/einaros/options.js/tree/master).